backend/copr_backend/background_worker_build.py

```diff
@@ -589,12 +605,19 @@ def _download_results(self):
         """
         Retry rsync-download the results several times.
         """
+        filter_ = None
+        if self.job.ssh_public_keys:
+            self.log.info("Builder allowed user SSH, not downloading the "
+                          "results for safety reasons.")
+            filter_ = ["+ success", "+ *.spec", "- *"]
```

Comment:
I'm thinking ... what if user replaces the remote side (server) `rsync` binary.  Can that trick the backend to download some unwanted stuff?

Ideas?

Reply:
This looks quite dangerous. Should we maybe verify some checksum of the binary, or something like that? Same for the `cat` commands mentioned above? But maybe that leads to the same issue (i.e. we can't be sure that the checksum tool wasn't tampered with)?